src/bootstrap/tool.rs

```diff
@@ -415,7 +415,7 @@ impl Step for Clippy {
     const ONLY_HOSTS: bool = true;
 
     fn should_run(run: ShouldRun) -> ShouldRun {
-        run.path("src/tools/clippy")
+       run.path("src/tools/clippy").default_condition(run.builder.build.config.extended)
```

Comment:
Compilation error. We don't have NLL in stage0 yet...

```
[00:01:24]    Compiling bootstrap v0.0.0 (file:///checkout/src/bootstrap)
[00:01:28] error[E0382]: use of moved value: `run.builder.build.config.extended`
[00:01:28]    --> /checkout/src/bootstrap/tool.rs:418:55
[00:01:28]     |
[00:01:28] 418 |        run.path("src/tools/clippy").default_condition(run.builder.build.config.extended)
[00:01:28]     |        --- value moved here                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ value used here after move
[00:01:28]     |
[00:01:28]     = note: move occurs because `run` has type `builder::ShouldRun<'_>`, which does not implement the `Copy` trait
```

Reply:
Also that one missing space should be re-inserted ;)